### ISSUE TITLE
Set retention_ Day = 30 days, but the actual retention time is still 7 days.

### ISSUE THREAD

**Issue by wenxin-01** [kubesphere/kubesphere#3609]
**Environment**
http://192.168.18.71:30880

**How To Reproduce**
1. Go to 'toolbox'
2. Click on 'Metering and Billing'
3. Click on 'view consumption' in Cluster resource consumption
4. view node 'i-0ks3sxr7' in cluster 'host'

**Actual behavior**
![image](https://user-images.githubusercontent.com/68640256/112741417-45a30c80-8fb8-11eb-9ce3-4d7135a40082.png)
![image](https://user-images.githubusercontent.com/68640256/112741429-61a6ae00-8fb8-11eb-92f7-372647b01f81.png)

/priority high
/kind bug
/assign @yunkunrao
/milestone 3.1.0

**Comment by yunkunrao** [comment]
This scenario occurs when metering's retention_day parameter was not consistent with prometheus retention_day. It's on the one hand. On the other hand, because metering only return data which currently backend stored. If prometheus retion_day is 7 days, metering return 7 days data at most.

**Comment by yunkunrao** [comment]
We will emphasize this part in kubespher doc.

**Comment by zryfish** [comment]
@yunkunrao So it's not a bug then?

**Comment by yunkunrao** [comment]
@zryfish  Yeah, I think it's not bug. Because retention_day parameter in metering config should be consistent with retention_day in prometheus config. In other words, if user want to change retention_day， users should modify prometheus crd other than metering config. And it make sense to me.

**Comment by zryfish** [comment]
OK. Better to have some documentation on that page.

**Comment by yunkunrao** [comment]
Got it.

**Comment by wenxin-01** [comment]
Close this issue after adding some documentation on that page.  /priority low
